src/server_name.rs

```diff
@@ -36,8 +36,9 @@ use std::error::Error as StdError;
 ///
 /// which will yield a `ServerName<'static>` if successful.
 ///
-/// // or, alternatively...
 ///
+/// or, alternatively...
```

Comment:
Would you mind keeping the empty lines as intended: one before, and one after "or, alternatively..."?

Reply:
IMO it is more idiomatic to spell this as `let x = ServerName::try_from("example.com").expect();`.

Reply:
I mean, thats what the above example is for, right?

Reply:
That's fair.